photon-core/src/main/java/org/photonvision/common/configuration/ConfigManager.java

```diff
@@ -68,16 +68,12 @@ public static ConfigManager getInstance() {
         if (INSTANCE == null) {
             Path rootFolder = PathManager.getInstance().getRootFolder();
             switch (m_saveStrat) {
-                case SQL:
-                    INSTANCE = new ConfigManager(rootFolder, new SqlConfigProvider(rootFolder));
-                    break;
-                case LEGACY:
-                    INSTANCE = new ConfigManager(rootFolder, new LegacyConfigProvider(rootFolder));
-                    break;
-                case ATOMIC_ZIP:
-                    // not yet done, fall through
-                default:
-                    break;
+                case SQL -> INSTANCE = new ConfigManager(rootFolder, new SqlConfigProvider(rootFolder));
+                case LEGACY ->
+                        INSTANCE = new ConfigManager(rootFolder, new LegacyConfigProvider(rootFolder));
+                case ATOMIC_ZIP -> {
+                    // TODO: Not done yet, fall through
```

Comment:
```suggestion
                    // TODO: Not done yet
```
this no longer falls through